ideascube/library/tests/test_migrations.py

```diff
@@ -0,0 +1,35 @@
+from ideascube.tests.helpers import MigrationTestCase
+
+
+section_int_to_name = {1: 'digital',
+               2: 'children-cartoons',
+               3: 'children-novels',
+               10: 'children-poetry',
+               11: 'children-theatre',
+               4: 'children-documentary',
+               5: 'children-comics',
+               6: 'adults-novels',
+               7: 'adults-documentary',
+               8: 'adults-comics',
+               12: 'adults-poetry',
+               13: 'adults-theatre',
+               9: 'game',
+               99: 'OTHER'}
```

Comment:
> If we reuse the migration data here, I'm fear we are testing almost nothing.

I disagree.

Look at #626 for example, where we realized the migration wasn't good. I'm going to reuse your helper to test things with various cases of data loaded into the db before the migration.

To me, the interesting part of the test is the data loaded into the db before the test runs the migration. That's what you want to verify after the migration. How the migration achieved that end result doesn't matter (from the point of view of the tests), as long as for all the cases you test the end data is good.

As such, if you feel you're « testing almost nothing » with the imported mapping, then you should feel the same if you're copy-pasting the mapping into the test. 😉 

But that doesn't matter much, I'm not going to block on this, I was merely wondering.